config/service_providers.yml

```diff
@@ -243,6 +252,15 @@ production:
       - 'https://sp-oidc-sinatra.<%= LoginGov::Hostdata.env %>.<%= LoginGov::Hostdata.domain %>/'
       <% end %>
 
+  'urn:gov:gsa:openidconnect:sp:expressjs':
+    agency: 'GSA'
+    cert: 'sp_expressjs_demo'
+    friendly_name: 'Example OIDC Client (Express.js)'
+    logo: '18f.svg'
+    redirect_uris:
+      - 'http://localhost:9393/'
+      - 'http://localhost:9393/auth/login-gov/callback'
+
```

Comment:
Thanks for working on this. I believe that for this to work in our lower environments, we have to register a new subdomain, such as `sp-expressjs.dev.login.gov`. Then, you can do this for the redirect_uris:
```yaml
- 'https://sp-expressjs.<%= LoginGov::Hostdata.env %>.<%= LoginGov::Hostdata.domain %>/'
- 'https://sp-expressjs.<%= LoginGov::Hostdata.env %>.<%= LoginGov::Hostdata.domain %>/auth/login-gov/callback'
```

Reply:
I'm not sure what all the requirements are to have the expressjs client up and running in our lower environments. I would ask in our devops Slack channel.